sdks/python/apache_beam/transforms/window.py

```diff
@@ -367,7 +381,10 @@ class FixedWindows(NonMergingWindowFn):
       range.
   """
 
-  def __init__(self, size, offset=0):
+  def __init__(self,
+               size,  # type: Union[int, float, Duration]
```

Comment:
Maybe make this into a named type? (Similarly for the Timestamp one.)

Reply:
done